enterprise/frontend/src/metabase-enterprise/moderation/index.js

```diff
@@ -36,3 +44,33 @@ export function getModerationStatusIcon(type) {
 export function getColor(type) {
   return getIn(ACTIONS, [type, "color"]);
 }
+
+export function getOpenIssues() {
```

Comment:
no clue what the final data structure will look like for this but please humor me for now